### PR TITLE
Adding intro faq

### PR DESCRIPTION
**Pull request by jameshupp**
## Changes
- Creates intro to cloud.gov content, giving us a place to point people who want more info about how cloud.gov works and might work for their agency work.
- Updates the docs.cloud.gov menu layout to encompass the intro copy.
## Review (I think I’m getting this right)

@jcscottiii 
@mogul 
@msecret 
## Notes
- As you’ll see, it’s not really formatted as an FAQ, since the questions were too long for the sidebar. We can take another approach if it turns out this is confusing, but now the titles are phrases.
- As we update the organization of this copy, changes will be faster and easier now that we’ve got the menu formatted for it.
- There’s a very minor CSS bug in the menu, small enough that it doesn’t disrupt anything and only appears in very specific circumstances. Happy to recreate if someone is working on the docs styling in the future, but for now I’m treating it like a non-issue.

### CODE REVIEW

**Comment by afeld** [comment]
I'd advocate for merging this sooner than later and iterating :innocent:

File: layouts/partials/menu.html [hunk 1]
```diff
@@ -7,25 +7,51 @@
...
-                <ul class="sub{{if $currentNode.HasMenuCurrent "main" . }} open{{end}}">
-                    {{ range .Children }}
-                    <li{{if $currentNode.IsMenuCurrent "main" . }} class="active"{{end}}><a href="{{.URL}}">{{ .Name }}</a> </li>
-                    {{ end }}
-                </ul>
-              {{end}}
+              <li class="sub-menu{{if $currentNode.HasMenuCurrent "main" . }} active{{end}}">
+                  {{ if .HasChildren }}
+                  <a href="javascript:;" class="">
+                      {{ .Pre }}
```
Comment (msecret) [inline layouts/partials/menu.html:13]: The indentation looks off on this line, I think it needs to be pulled one space to the left.

File: layouts/partials/menu.html [hunk 2]
```diff
@@ -7,25 +7,51 @@
...
+		      <span>{{ .Name }}</span>
+                      <span class="menu-arrow fa {{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }}fa-angle-down{{else}}fa-angle-right{{end}}"></span>
```
Comment (msecret) [inline layouts/partials/menu.html:15]: same here.

File: layouts/partials/menu.html [hunk 3]
```diff
@@ -7,25 +7,51 @@
...
+                  </a>
+                  <ul class="sub nested-menu {{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }} open {{end}}" style="{{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }} display: block; {{else}}display: none; {{end}}">
+                      {{ range .Children }}
```
Comment (msecret) [inline layouts/partials/menu.html:18]: Actually looks like theres a bunch of indentation issues in this file. I think it should be a tab of two spaces so example:

```
<div>
  <div>
    {{ if.whatever }}
    <ul>
    </ul>
  </div>
</div>
```

File: static/js/scripts.js [hunk 4]
```diff
@@ -1,283 +1,281 @@
...
-
-	})();
-})(jQuery);
-
-jQuery(document).ready(function(){
-    initializeJS();
-    $('[rel=show-github]').showGithub();
-});
-
+function initializeJS() {
```
Comment (msecret) [inline static/js/scripts.js:1]: It looks like some of this code came from outside? Will this code need to be changed a lot? I kinda feel like we should just leave it as is even if it's a little messy.

**Comment by msecret** [comment]
I commented on the code, I'm fine with committing this change now, even with the indentation issues.

**Comment by mogul** [comment]
This is good to go for me... It needs a real content pass for consistent voice, breaking up/consolidating points, etc. but even in the current form this will go miles toward knocking down our workload from redundant questions. :ship: :shipit: !

**Pull request merged**